infra/blueprint-test/pkg/git/git.go

```diff
@@ -0,0 +1,110 @@
+package git
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/mitchellh/go-testing-interface"
+)
+
+type CmdCfg struct {
+	gitBinary   string         // git binary
+	dir         string         // dir to execute commands in
+	commitFlags []string       // additional flags when running git commit
```

Comment:
Since these are specific to the CommitWithMsg method, this should be moved to simply being a parameter of that function.

Reply:
done